### ISSUE TITLE
Show Toast not working

### ISSUE THREAD

**Issue by dleland** [alexed1/LightningFlowComponents#178]
I have been using the ShowToast component for a while now but I can no longer get the message to show.  I typically have the ShowToast as the last step of my flow, to show a completion message.

**Comment by gvgramazio** [comment]
I have this issue too.

**Comment by brainage2** [comment]
How do i show an error message in an autolaunched flow triggered by Process builder? Any workarounds?

**Comment by ericrsmith35** [comment]
Please try [Version 2.0](https://unofficialsf.com/show-toast-flow-action/) and let us know if you are still having issues.

**Comment by lvillanueva1916** [comment]
Currently having this problem as well. Verified version 2.0 is installed. I've tried plugging in the component in several different places in the flow, but still not working

**Comment by ericrsmith35** [comment]
![Capture](https://user-images.githubusercontent.com/24724447/76526129-72cf5c00-6443-11ea-87d9-adaf7bd383aa.PNG)
Try creating a very simple test flow and add it to a Lightning Page and see if you are still having issues.  I am unable to reproduce your problem.

**Comment by lvillanueva1916** [comment]
I think maybe I'm misunderstanding how this has to be used - I was trying to use it in a flow that was triggered from a button press, which was really a url redirect. This looks like it has to be embedded in the page?

**Comment by ericrsmith35** [comment]
No, it is an action that can be added to a flow.  See if it makes any
difference if you have it on a Screen Flow rather than an Autolaunched Flow.

**Comment by lvillanueva1916** [comment]
That's just it, though, I put it in a screen flow initially. See screen shot below.

![image](https://user-images.githubusercontent.com/32394760/76534956-2c303080-6448-11ea-98c9-204d9698abd9.png)

**Comment by adwhite75** [comment]
@lvillanueva1916  / @gvgramazio is this still an issue you're seeing? To be clear, this will not work in the Debugger you have to test it out on a lightning page.

**Comment by narmic** [comment]
Hi @ericrsmith35 I created a screen flow using ShowToast 2.0 with a clickable link (using the same structure as the fContactRecordURL example) and added the flow to a lightning page in a sandbox. Works great an a desktop however when I test it on in the mobile app the {url} piece is replaced with "{1}" rather than a clickable link. Is there a way to make the link clickable in the mobile app as well? Not 100% sure if this is expected or not. Thanks for creating so many cool flow tools!

**Comment by ericrsmith35** [comment]
The '{url}' is replaced with '{1}' inside of the component, but the system then should replace that with one of the other attributes. 
 I'm sorry, but I do not know why the exact same component would work on a desktop but not on mobile.  Maybe someone else here could take a look?
